### PR TITLE
Business Object server builds

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/live-windows-virtual-machines/bus_obj1.tf [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+      volume_type           = var.volume_type
+      encrypted             = var.ebs_encrypted
+      kms_key_id            = data.aws_kms_key.ebs.arn
+    }
+  ]
+
+  ebs_block_device = [
+    {
+      delete_on_termination = var.delete_on_termination
+      device_name           = "/dev/xvdg"
```
Comment (mattystone) [inline groups/live-windows-virtual-machines/bus_obj1.tf:299]: ebs_block_device's have conflicting names?

File: groups/live-windows-virtual-machines/profiles/heritage-live-eu-west-2/vars [hunk 2]
```diff
@@ -95,4 +95,19 @@ chips_estor_cw_logs = {
    "system" = {
     log_group_retention = 365
   }
+}
+
+# Business Objects Server 1
+bus_obj_1_application       = "Business Objects Server 1"
+bus_obj_1_ec2_name          = "busobj1-app"
+bus_obj_1_ec2_instance_size = "m5.2xlarge"
+bus_obj_1_ami               = "ami-082ace3b4ab9d3345"
```
Comment (mattystone) [inline groups/live-windows-virtual-machines/profiles/heritage-live-eu-west-2/vars:104]: static ami, should this be from data sources with filters?

File: groups/live-windows-virtual-machines/bus_obj1.tf [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+      "ServiceTeam", var.ServiceTeam
+    )
+  )
+}
+
+# ------------------------------------------------------------------------------
+# Business Objects Server 1 EC2
+# ------------------------------------------------------------------------------
+
+module "bus_obj_1_ec2" {
```
Comment (mattystone) [inline groups/live-windows-virtual-machines/bus_obj1.tf:270]: No userdata passed in for config e.g. cw logs. These being configured manually post build?

File: groups/live-windows-virtual-machines/bus_obj1.tf (hunk 1 above)
Comment (willwatters) [reply groups/live-windows-virtual-machines/bus_obj1.tf:299]: Corrected:

ebs_block_device = [
    {
      delete_on_termination = var.delete_on_termination
      device_name           = "/dev/xvdf"
      encrypted             = var.ebs_encrypted
      volume_size           = "80"
      volume_type           = var.volume_type
      kms_key_id            = data.aws_kms_key.ebs.arn
    },
    {
      delete_on_termination = var.delete_on_termination
      device_name           = "/dev/xvdg"
      encrypted             = var.ebs_encrypted
      volume_size           = "150"
      volume_type           = var.volume_type
      kms_key_id            = data.aws_kms_key.ebs.arn
    },
    {
      delete_on_termination = var.delete_on_termination
      device_name           = "/dev/xvdh"
      encrypted             = var.ebs_encrypted
      volume_size           = "40"
      volume_type           = var.volume_type
      kms_key_id            = data.aws_kms_key.ebs.arn
    }
  ]

File: groups/live-windows-virtual-machines/bus_obj1.tf (hunk 3 above)
Comment (willwatters) [reply groups/live-windows-virtual-machines/bus_obj1.tf:270]: These are windows servers so will be configured manually post build.

File: groups/live-windows-virtual-machines/profiles/heritage-live-eu-west-2/vars (hunk 2 above)
Comment (willwatters) [reply groups/live-windows-virtual-machines/profiles/heritage-live-eu-west-2/vars:104]: Not use base AMI, using Windows Server 2019 AMI.

**Review by mattystone — APPROVED** [review]

**Pull request merged**